dataflow/src/main/java/org/checkerframework/dataflow/cfg/CFGBuilder.java

```diff
@@ -394,15 +394,16 @@ public String toString() {
          * Map from exception type to labels of successors that may be reached as a result of that
          * exception.
          */
-        protected final Map<TypeMirror, Set<Label>> exceptions;
+        protected final Map<TypeMirror, Set<Pair<TypeMirror, Label>>> exceptions;
```

Comment:
Please update the documentation of the field and explain what the Pairs represent.